<launch>
  <node name="ekf_node" pkg="state_estimation" type="ekf_fixed_rel_node" respawn="false" output="screen" clear_params="true">
    <remap from="imu" to="/x8/output/imu" />
    <remap from="rel_pose" to="/vicon_pose_cov" />
    <remap from="ekf_node/pose" to="x8/output/pose" />
    <remap from="ekf_node/velocity" to="x8/output/velocity" />
    
    <param name="sensor_offset_source" type="string" value="file" />
    <param name="relative_pose_sensor_file" type="string" value="$(find state_estimation)/calibration/vicon_pose.yaml" />
    <param name="rel_world_file" type="string" value="$(find state_estimation)/calibration/vicon_world.yaml" />
    
  </node>
</launch>
